### DISCUSSION TITLE
Does this library work with React Native too?

### DISCUSSION THREAD

**Discussion by gabimoncha** [percy/percy-storybook#388] [Support]
I searching to implement visual testing for React Native and found that percy.io is a good platform for this.
But I haven't found anything on the docs regarding React Native

**Comment by Robdel12** [comment]
Hey @gabrielmoncea! Probably not -- Percy only works with web based platforms (we capture the DOM state and then rerender that DOM in our browsers). We don't have any first-class support for native applications. As a workaround, you could capture screenshots in your native apps E2E tests and then upload those images to Percy using the CLI upload command: https://docs.percy.io/docs/cli-upload

**Comment by dawidk92** [comment]
Hey @gabimoncha 👋

You might want to check out Sherlo - I built it for visual testing React Native Storybook. Could be helpful!

https://github.com/sherlo-io/sherlo

**Reply by gabimoncha** [reply]
already starred, but it would be nice to have a locally hosted free tier

**Reply by dawidk92** [reply]
Thanks for starring! 🙏

No self-hosted plans, but Sherlo is fully free while in beta (not for much longer though - we're pretty stable now).
We'll have a free tier post-beta, but if you want unlimited access, now is the perfect time to try it out! 🚀

You can simply sign up at https://app.sherlo.io/ and start testing right away!
